include/sway/server.h

```diff
@@ -98,6 +99,8 @@ struct sway_server {
 	struct wlr_xdg_activation_v1 *xdg_activation_v1;
 	struct wl_listener xdg_activation_v1_request_activate;
 
+	struct wl_list mirrors; // sway_mirror::link
```

Comment:
Mutated by `mirror_create` and `mirror_destroy`.

Accessed by `mirror_destroy_all`.